components/net/http_loader.rs

```
@@ -603,18 +603,17 @@ fn send_request_to_devtools(msg: ChromeToDevtoolsControlMsg,
     devtools_chan.send(DevtoolsControlMsg::FromChrome(msg)).unwrap();
 }
 
-fn send_response_to_devtools(devtools_chan: Option<Sender<DevtoolsControlMsg>>,
+fn send_response_to_devtools(devtools_chan: &Sender<DevtoolsControlMsg>,
                              request_id: String,
                              headers: Option<Headers>,
                              status: Option<RawStatus>,
                              pipeline_id: PipelineId) {
-    if let Some(ref chan) = devtools_chan {
-        let response = DevtoolsHttpResponse { headers: headers, status: status, body: None, pipeline_id: pipeline_id };
-        let net_event_response = NetworkEvent::HttpResponse(response);
+    let response = DevtoolsHttpResponse { headers: headers, status: status, body: None, pipeline_id: pipeline_id };
+    let net_event_response = NetworkEvent::HttpResponse(response);
+
+    let msg = ChromeToDevtoolsControlMsg::NetworkEvent(request_id, net_event_response);
+    devtools_chan.send(DevtoolsControlMsg::FromChrome(msg)).unwrap();
```

@jdm Is it better to fail silently or `unwrap` here?